public/js/index.js

```diff
@@ -9,167 +9,13 @@
 console.info(`rq_ui version ${RQ_PARAMS.VERSION} starting`)
 console.info(`rq_ui config format version ${RQ_PARAMS.CONFIG_FORMAT_VERSION}`)
 
-/**
- * Extends jQuery. To be called on a jQuery element of class 'widget'.
- *
- *  @returns {object} - a complete widget configuration object.
- */
-jQuery.fn.getWidgetConfiguration = function () {
-  return this.data(RQ_PARAMS.WIDGET_NAMESPACE)
-}
-
 const keyControl = new KeyControl('#keyControl')
-const servoConfig = new ServoConfig()
```

Comment:
Line 22 should stay in index.js

Reply:
Yup. See my suggestions in [Slack](https://questinstitutegroup.slack.com/archives/D05QFQ6FPJP/p1698970298402959?thread_ts=1698556719.024249&cid=D05QFQ6FPJP)